src/Python/qsharp/tests/test_azure.py

```diff
@@ -74,7 +74,7 @@ def test_workspace_create_with_resource_id():
     resourceGroupName = "test"
     workspaceName = "WorkspaceNameWithMockProviders"
     targets = qsharp.azure.connect(
-        resourceId=f"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/Workspaces/{workspaceName}")
+        resourceId=f"/subscriptions/{subscriptionId}/RESOurceGroups/{resourceGroupName}/providers/Microsoft.Quantum/Workspaces/{workspaceName}")
```

Comment:
Was this an accidental change?

Reply:
no, it's a test to make sure it's not case sensitive.

Reply:
No, it's to test the case-insensitivity. 😊 I'll add a comment so that's clear, thanks for questioning.